src/Dfe.PlanTech.Web/Controllers/QuestionsController.cs

```diff
@@ -97,7 +98,16 @@ public async Task<IActionResult> GetNextUnansweredQuestion(string sectionSlug,
         {
             // Remove the current invalid submission and redirect to self-assessment page
             await deleteCurrentSubmissionCommand.DeleteCurrentSubmission(section, cancellationToken);
-            TempData["SubtopicError"] = configuration["ErrorMessages:ConcurrentUsersOrContentChange"];
+
+            var contactLink = await _getEntityFromContentfulQuery.GetEntityById<NavigationLink>(configuration["ContactUs:LinkId"]);
```

Comment:
Would be good to move the `configuration` out of here, create classes/records to store the configurations, and bind/create them in the dependency injection setup. Especially the error messages, given how many of those and where they're used.

Reply:
Good idea, is this more the kind of thing you had in mind?